LeaderboardBackend/Controllers/CategoriesController.cs

```diff
@@ -30,14 +30,15 @@ public async Task<ActionResult<Category>> GetCategory(long id)
 	{
 		// NOTE: Should this use [AllowAnonymous]? - Ero
 
-		Category? category = await _categoryService.GetCategory(id);
-
-		if (category == null)
+		try
+		{
+			Category category = await _categoryService.Get(id);
+			return Ok(category);
+		}
+		catch (System.Exception)
```

Comment:
Would it be better not to use exceptions for control flow, and instead only use them when something actually goes wrong? The category requested not being found is just something that can happen here, not really an error.

Reply:
Two options here:
1. The underlying service DB call throws an `ArgumentNullException` if the DB query turns up zilch. I just didn't add a catch block for that specifically, but I was lazy. That being said;
2. There _is_ another method I can call (e.g. `FirstOrDefault()`) which'll return `null` instead that I could use, tho that will involve me rewriting all services' Get calls, for consistency. So heads up on that.